### ISSUE TITLE
Formal syntax doesn't show values from other spces

### ISSUE THREAD

**Issue by OnkarRuikar** [mdn/rari#166]
### Summary

Formal syntax on CSS sizing property `width`, `max-width`, `min-width` etc. pages do not accumulate values from all the specs.

### URL

https://developer.mozilla.org/en-US/docs/Web/CSS/width#formal_syntax

### Reproduction steps

1. Open page https://developer.mozilla.org/en-US/docs/Web/CSS/width#formal_syntax
2. Observe the Formal Syntax section
3. Then observe BCD table at the bottom
4. The BCD table lists values like `stretch` and `anchor-size()` which are not there in the Formal Syntax.

### Expected behavior

In sizing properties cases, the formal syntaxes should accumulate value syntaxes from all the spec webref jsons:
- [css-sizing](https://github.com/w3c/webref/blob/cd570c8e8fa7a0351babc56e0a84254885378c2c/ed/css/css-sizing.json#L10)
- [css-sizing-4](https://github.com/w3c/webref/blob/4b979577849548279e3fdafca2486911ddf10efd/ed/css/css-sizing-4.json#L9)
- [css-anchor-position-1](https://github.com/w3c/webref/blob/4b979577849548279e3fdafca2486911ddf10efd/ed/dfns/css-anchor-position-1.json#L1847)

Also, in the case of all other feature syntaxes, accumulate value syntaxes from all the available specs jsons.

### Actual behavior

The Formal Syntax section shows only the syntax from `css-sizing.json`.

### Anything else?

I've raised [an issue](https://github.com/w3c/webref/issues/1312) in Webref to map sizing properties to the `anchor-size()` value.

### Validations

- [X] I have read the [Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/).
- [X] I have verified that there isn't [already an issue](https://github.com/mdn/yari/issues) that reports the same bug to avoid creating a duplicate.
- [X] I have checked that this is a concrete bug. For Q&A open a [GitHub Discussion](https://github.com/orgs/mdn/discussions/categories/platform).

**Comment by bsmth** [comment]
@OnkarRuikar is this done now?

**Comment by OnkarRuikar** [comment]
@bsmth no, the width syntax is still not including [css-sizing-4](https://github.com/w3c/webref/blob/8ec88e04bc2f008ef34e681d7f763bff9aaaab66/ed/css/css-sizing-4.json#L12) webref. The width formal syntax needs to include `stretch`.
